### PR TITLE
Added missing game info

### PR DESCRIPTION
**Pull request by AquilaSagitta**
When user is in game these fields will be populated with relevant data allowing you to check if they are actively playing a game or not

### CODE REVIEW

File: src/structures/PlayerSummary.js [hunk 1]
```diff
@@ -21,6 +21,10 @@ class PlayerSummary extends Player {
 		this.countryCode = player.loccountrycode;
 		this.stateCode = player.locstatecode;
 		this.cityID = player.loccityid;
+		this.gameserverip = player.gameserverip;
+        	this.gameserversteamid = player.gameserversteamid;
+        	this.gameextrainfo = player.gameextrainfo;
+        	this.gameid = player.gameid;
```
Comment (xDimGG) [inline src/structures/PlayerSummary.js]:
```suggestion
        	this.gameID = player.gameid;
```

File: src/structures/PlayerSummary.js [hunk 2]
```diff
@@ -21,6 +21,10 @@ class PlayerSummary extends Player {
 		this.countryCode = player.loccountrycode;
 		this.stateCode = player.locstatecode;
 		this.cityID = player.loccityid;
+		this.gameserverip = player.gameserverip;
+        	this.gameserversteamid = player.gameserversteamid;
+        	this.gameextrainfo = player.gameextrainfo;
```
Comment (xDimGG) [inline src/structures/PlayerSummary.js]:
```suggestion
        	this.gameExtraInfo = player.gameextrainfo;
```

File: src/structures/PlayerSummary.js [hunk 3]
```diff
@@ -21,6 +21,10 @@ class PlayerSummary extends Player {
 		this.countryCode = player.loccountrycode;
 		this.stateCode = player.locstatecode;
 		this.cityID = player.loccityid;
+		this.gameserverip = player.gameserverip;
+        	this.gameserversteamid = player.gameserversteamid;
```
Comment (xDimGG) [inline src/structures/PlayerSummary.js]:
```suggestion
        	this.gameServerID = player.gameserversteamid;
```

File: src/structures/PlayerSummary.js [hunk 4]
```diff
@@ -21,6 +21,10 @@ class PlayerSummary extends Player {
 		this.countryCode = player.loccountrycode;
 		this.stateCode = player.locstatecode;
 		this.cityID = player.loccityid;
+		this.gameserverip = player.gameserverip;
```
Comment (xDimGG) [inline src/structures/PlayerSummary.js]:
```suggestion
		this.gameServerIP = player.gameserverip;
```

File: src/structures/PlayerSummary.js (hunk 1 above)
Comment (xDimGG) [inline src/structures/PlayerSummary.js]: I would prefer if these four properties were in a `this.game` object; furthermore, they should be camel cased.

Comment (AquilaSagitta) [reply src/structures/PlayerSummary.js]: These properties directly relate to the player revealing which game and server that player is using currently

**Pull request merged**